common/src/chain/config/mod.rs

```diff
@@ -452,6 +457,60 @@ fn create_mainnet_genesis() -> Genesis {
     )
 }
 
+fn decode_hex<T: serialization::DecodeAll>(hex: &str) -> T {
+    let bytes = Vec::from_hex(hex).expect("Hex decoding shouldn't fail");
+    <T as serialization::DecodeAll>::decode_all(&mut bytes.as_slice())
+        .expect("Decoding shouldn't fail")
+}
+
+fn create_testnet_genesis() -> Genesis {
+    // TODO: use coin_decimals instead of a fixed value
+    const COIN: Amount = Amount::from_atoms(100000000000);
+
+    let total_amount = (COIN * 100_000_000).expect("must be valid");
+    let initial_pool_amount = (COIN * 40_000).expect("must be valid");
+    let mint_output_amount = (total_amount - initial_pool_amount).expect("must be valid");
+
+    let genesis_message = String::new();
+
+    let genesis_mint_destination = decode_hex::<PublicKey>(
+        "00027a9771bbb58170a0df36ed43e56490530f0f2f45b100c42f6f405af3ef21f54e",
+    );
+    let decommission_pub_key = decode_hex::<PublicKey>(
+        "0002ea30f3bb179c58022dcf2f4fd2c88685695f9532d6a9dd071da8d7ac1fe91a7d",
+    );
+    let staker_pub_key = decode_hex::<PublicKey>(
+        "0002884adf48b0b32ab3d66e1a8b46576dfacca5dd25b66603650de792de4dd2e483",
+    );
+
+    let vrf_pub_key = decode_hex::<VRFPublicKey>(
+        "00d4b8138be57c1bddfff397ffc72ca0453131f8df364d2b6bf4a8c6558ed81d79",
+    );
+
+    let mint_output = TxOutput::Transfer(
+        OutputValue::Coin(mint_output_amount),
+        Destination::PublicKey(genesis_mint_destination),
+    );
+
+    let initial_pool = TxOutput::CreateStakePool(
+        H256::zero().into(),
+        Box::new(StakePoolData::new(
+            initial_pool_amount,
+            Destination::PublicKey(staker_pub_key),
+            vrf_pub_key,
+            Destination::PublicKey(decommission_pub_key),
+            PerThousand::new(10).expect("must be valid"),
+            (COIN * 100).expect("must be valid"),
```

Comment:
I think that entire reward should go to stake owner, so that no one would be tempted to delegate to this pool
```
 PerThousand::new(1000).expect("must be valid"),
 Amount::ZERO,
```

Reply:
Why is delegating to this pool a problem?

Reply:
easy money - it's guarantee to be the winner for the first 2 epochs

Reply:
I don't think that's true, because there's no special treatment for this pool.

Reply:
its existence already makes it special: it's the only pool that can mine 2 epochs. It's just looks safer to me and also better than arbitrary numbers. But if you think that's invalid in this case I won't insist.